server/notification-providers/google-chat.js

```diff
@@ -16,26 +15,74 @@ class GoogleChat extends NotificationProvider {
         try {
             // Google Chat message formatting: https://developers.google.com/chat/api/guides/message-formats/basic
 
-            let textMsg = "";
-            if (heartbeatJSON && heartbeatJSON.status === UP) {
-                textMsg = "✅ Application is back online\n";
-            } else if (heartbeatJSON && heartbeatJSON.status === DOWN) {
-                textMsg = "🔴 Application went down\n";
-            }
+            let chatHeader = {
+                title: "Uptime Kuma Alert",
+            };
 
-            if (monitorJSON && monitorJSON.name) {
-                textMsg += `*${monitorJSON.name}*\n`;
+            if (monitorJSON && heartbeatJSON) {
+                chatHeader["title"] =
+                    heartbeatJSON["status"] === UP
+                        ? `✅ ${monitorJSON["name"]} is back online`
+                        : `🔴 ${monitorJSON["name"]} went down`;
             }
 
-            textMsg += `${msg}`;
+            // always show msg
+            let sectionWidgets = [
+                {
+                    textParagraph: {
+                        text: `<b>Message:</b>\n${msg}`,
+                    },
+                },
+            ];
 
+            // add time if available
+            if (heartbeatJSON) {
+                sectionWidgets.push({
+                    textParagraph: {
+                        text: `<b>Time (UTC):</b>\n${heartbeatJSON["localDateTime"]}`,
+                    },
+                });
+            }
+
+            // add button for monitor link if available
             const baseURL = await setting("primaryBaseURL");
-            if (baseURL && monitorJSON) {
-                textMsg += `\n${baseURL + getMonitorRelativeURL(monitorJSON.id)}`;
+            if (baseURL) {
+                sectionWidgets.push({
+                    buttonList: {
+                        buttons: [
+                            {
+                                text: "Visit Uptime Kuma",
+                                onClick: {
+                                    openLink: {
+                                        url:
+                                            baseURL + (monitorJSON ? getMonitorRelativeURL(
+                                                monitorJSON.id
+                                            ) : "/"),
```

Comment:
nit:Extracting this might be more readable
```suggestion
                const urlPath = monitorJSON ? getMonitorRelativeURL(monitorJSON.id) : "/"
                sectionWidgets.push({
                    buttonList: {
                        buttons: [
                            {
                                text: "Visit Uptime Kuma",
                                onClick: {
                                    openLink: {
                                        url: baseURL + urlPath,
```